app/Http/Controllers/SlaReportController.php

```diff
@@ -137,7 +138,9 @@ public function slaReport(Request $request)
         }
 
         $tickets = $tickets->where('conversations.threads_count', '!=', '0')->get();
+        $user=new UsersController();
```

Comment:
Please do not use one controller inside another controller. Please define the method inside the controller itself.